### PR TITLE
Add cluster-density-v2 Rosa tests

### PR DESCRIPTION
**Pull request by svetsa-rh**
Add new tests for cluster-density-v2 for classic Rosa testing

### CODE REVIEW

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-cluster-density-v2-images pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-cluster-density-v2-ocp-qe-perfscale-ci-tests-rosa-cluster-density-v2

**Comment by svetsa-rh** [comment]
@paigerube14 Can you ptal and approve?

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-nightly-cluster-density-v2.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      memory: 200Mi
+tests:
+- as: ocp-qe-perfscale-ci-tests-rosa-cluster-density-v2
+  steps:
+    cluster_profile: aws-sd-qe
+    env:
+      CHANNEL_GROUP: nightly
+      OPENSHIFT_VERSION: "4.13"
+    test:
+    - chain: openshift-qe-cluster-density-v2
```
Comment (paigerube14) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-nightly-cluster-density-v2.yaml:31]: I think you should be able to use "ref" here but can keep it as chain

**Comment by svetsa-rh** [comment]
@paigerube14 I see that it still needs lgtm label. Can you add it? TIA!

**Pull request merged**